ios/bigtable/pom.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
OK, I tried to "mimic" runners, sdks where we have a 's'. However, no problem to use io instead.

For io.gcp.bigtable, I guess you talk about the package to use, right ?